Tensile/SolutionStructs.py

```diff
@@ -3302,11 +3356,21 @@ def assignDerivedParameters(state):
         state["DirectToLdsA"] = True
         state["LocalWriteUseSgprA"] = True
         #print("DirectToLdsA", state["DirectToLdsA"])
+        # DorectToLsdA + DirectToVgprA does not work. Not enable DorectToLsdA if DirectToVgprA is true
+        if state["DirectToVgprA"]:
+          state["DirectToLdsA"] = False
+          state["LocalWriteUseSgprA"] = False
+          print2("DirectToLdsA is disabled because DirecttoVgprA is true")
```

Comment:
capitalize the T in "to" for DirecttoVgprA

Reply:
I corrected this to "To".